importers/djornl/parser.py

```diff
@@ -0,0 +1,213 @@
+                    'edge_type': edge_remap[edge_type],
+                })
+
+        return {
+            'nodes': [{'_key': n} for n in node_ix.keys()],
+            'edges': edges,
+        }
+
+
+    def load_node_metadata(self):
+        """Load node metadata"""
+
+        nodes = []
+        expected_col_count = self.config()['_NODE_FILE_COL_COUNT']
+        with open(self.config()['_NODE_PATH']) as fd:
+            csv_reader = csv.reader(fd, delimiter=',')
+            next(csv_reader, None)  # skip headers
+            line_no = 1
+            for row in csv_reader:
+                line_no += 1
+
+                cols = [c.strip() for c in row]
+                if len(cols) != expected_col_count:
+                    n_cols = len(cols)
+                    raise RuntimeError(f"line {line_no}: expected {expected_col_count} cols, found {n_cols}")
+
+                _key = cols[0]
+                node_type = cols[1]
+                if node_type != 'gene' and node_type != 'pheno':
+                    raise RuntimeError(f"line {line_no}: invalid node type: {node_type}")
+
+                go_terms = [c.strip() for c in cols[10].split(',')] if len(cols[10]) else []
+
+                doc = {
+                    '_key': _key,
+                    'node_type': node_type,
+                    'transcript': cols[2],
+                    'gene_symbol': cols[3],
+                    'gene_full_name': cols[4],
+                    'gene_model_type': cols[5],
+                    'tair_computational_desc': cols[6],
+                    'tair_curator_summary': cols[7],
+                    'tair_short_desc': cols[8],
+                    'go_descr': cols[9],
+                    'go_terms': go_terms,
+                    'mapman_bin': cols[11],
+                    'mapman_name': cols[12],
+                    'mapman_desc': cols[13],
+                    'pheno_aragwas_id': cols[14],
+                    'pheno_desc1': cols[15],
+                    'pheno_desc2': cols[16],
+                    'pheno_desc3': cols[17],
+                    'pheno_ref': cols[18],
+                    'user_notes': cols[19],
+                }
+                nodes.append(doc)
+
+        return {'nodes': nodes}
+
+
+    def load_cluster_data(self):
+        """Annotate genes with cluster ID fields."""
+        nodes = []
+        cluster_paths = self.config()['_CLUSTER_PATHS']
+        for (cluster_label, path) in cluster_paths.items():
+            with open(path) as fd:
+                csv_reader = csv.reader(fd, delimiter='\t')
+                for row in csv_reader:
+                    if len(row) > 1:
+                        # remove the 'Cluster' text
+                        cluster_id = row[0].replace('Cluster','')
+                        gene_keys = row[1:]
+                        nodes += [
+                            {'_key': key, cluster_label: int(cluster_id)}
+                            for key in gene_keys
+                        ]
```

Comment:
Changed this slightly so that instead of having the "Cluster_n_" text repeated, we just use an integer